template/frameworks/ava/babel.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Adding `babel.config.js` may have conflicts if other frameworks also needs it.

Can we put the config in `babel` of `ava.config.js` ? https://github.com/avajs/ava/blob/master/docs/recipes/babel.md#customize-how-ava-compiles-your-test-files

Reply:
- [x] Fixed in 4c26ca1